modules/plugins/enigmail.js

```diff
@@ -106,29 +106,23 @@ let Log = setupLogging("Conversations.Modules.Enigmail");
 let window = getMail3Pane();
 let hasEnigmail;
 try {
-  /* globals EnigmailCommon, EnigmailCore, EnigmailData, EnigmailLocale,
+  /* globals EnigmailCore, EnigmailData, EnigmailLocale,
              EnigmailFuncs, Enigmail, EnigmailPrefs, EnigmailDialog,
              EnigmailConstants, EnigmailRules, EnigmailArmor,
-             EnigmailDecryption */
-  ChromeUtils.import("resource://enigmail/core.jsm");
-  ChromeUtils.import("resource://enigmail/data.jsm");
-  ChromeUtils.import("resource://enigmail/dialog.jsm");
-  ChromeUtils.import("resource://enigmail/prefs.jsm");
-  ChromeUtils.import("resource://enigmail/locale.jsm");
+             EnigmailDecryption, EnigmailExecution */
+  ChromeUtils.import("chrome://enigmail/content/modules/core.jsm");
+  ChromeUtils.import("chrome://enigmail/content/modules/data.jsm");
+  ChromeUtils.import("chrome://enigmail/content/modules/dialog.jsm");
+  ChromeUtils.import("chrome://enigmail/content/modules/prefs.jsm");
+  ChromeUtils.import("chrome://enigmail/content/modules/locale.jsm");
+  ChromeUtils.import("chrome://enigmail/content/modules/constants.jsm");
+  // eslint-disable-next-line no-unused-vars
+  ChromeUtils.import("chrome://enigmail/content/modules/execution.jsm");
   hasEnigmail = true;
   Log.debug("Enigmail plugin for Thunderbird Conversations loaded!");
 } catch (e) {
   hasEnigmail = false;
-  Log.debug("Enigmail is older than 1.9 or doesn't seem to be installed...");
-  try {
-    ChromeUtils.import("resource://enigmail/enigmailCommon.jsm");
-    ChromeUtils.import("resource://enigmail/commonFuncs.jsm");
-    hasEnigmail = true;
-    Log.debug("Enigmail plugin for Thunderbird Conversations loaded!");
-  } catch (e) {
-    hasEnigmail = false;
-    Log.debug("Enigmail doesn't seem to be installed...");
-  }
+  Log.debug("Enigmail doesn't seem to be installed...");
```

Comment:
I'd suggest adding this here, so that we don't get extra warnings on TB 60 builds:
```suggestion
  Log.debug("Enigmail doesn't seem to be installed...");
  var EnigmailConstants = {};
```

Reply:
Fixed differently.
Move `nsIEnigmail = EnigmailConstants;` into `if (hasEnigmail) {`.